test/Rx/Functional/Operator/IsEmptyTest.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace Rx\Functional\Operator;
+
+use Rx\Functional\FunctionalTestCase;
+
+class IsEmptyTest extends FunctionalTestCase
+{
+
+    /**
+     * @test
+     */
+    public function should_return_true_if_source_is_empty()
+    {
+        $xs = $this->createHotObservable([
+            onCompleted(300)
```

Comment:
A similar situation is in [should_return_false_if_source_emits_element](https://github.com/ReactiveX/RxPHP/pull/125/files/d78d3a3d84bce819ac0e936f8b73f8a1bc4b7574#diff-b99fb56076d5d32fbf2472d605dbf506R36) but I added a separate test for this.